example/go.mod

```diff
@@ -1,17 +1,141 @@
 module github.com/docker/cli-docs-tool/example
 
-go 1.16
+go 1.18
 
 require (
-	github.com/docker/buildx v0.6.3
-	github.com/docker/cli v20.10.7+incompatible
-	github.com/docker/cli-docs-tool v0.0.0
-	github.com/spf13/cobra v1.2.1
+	github.com/docker/buildx v0.8.2
+	github.com/docker/cli v20.10.12+incompatible
```

Comment:
perhaps change this to the latest 20.10.x (I know it's not used because of the `replace`, but 🤷‍♂️)

Reply:
oh, this is the example .. perhaps doesn't matter